{"uid":"c9f4c65a71a3d09","name":"test_should_return_403_if_access_denied","fullName":"tests.api.test_api_signup#test_should_return_403_if_access_denied","historyId":"e4e5638c4829e59acca99c47addb7ed0","time":{"start":1718383432713,"stop":1718383432726,"duration":13},"status":"failed","statusMessage":"AssertionError: Expected status code 403\nassert 400 == 403\n +  where 400 = <Response [400]>.status_code\n +    where <Response [400]> = HTTPError('400 Client Error:  for url: http://localhost:4001/users/signup').response","statusTrace":"sign_up_api = <api.post_sign_up.SignUp object at 0x7ff6a1768f20>\n\n    def test_should_return_403_if_access_denied(sign_up_api: SignUp):\n        user = get_random_user()\n        user.token = \"invalid_token\"  # Simulate an invalid token or unauthorized access\n        try:\n>           sign_up_api.api_call(user)\n\ntests/api/test_api_signup.py:56: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \napi/post_sign_up.py:7: in api_call\n    response = self.make_request(\"POST\", \"users/signup\", json=user_data)\napi/base_api.py:33: in make_request\n    response.raise_for_status()\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <Response [400]>\n\n    def raise_for_status(self):\n        \"\"\"Raises :class:`HTTPError`, if one occurred.\"\"\"\n    \n        http_error_msg = \"\"\n        if isinstance(self.reason, bytes):\n            # We attempt to decode utf-8 first because some servers\n            # choose to localize their reason strings. If the string\n            # isn't utf-8, we fall back to iso-8859-1 for all other\n            # encodings. (See PR #3538)\n            try:\n                reason = self.reason.decode(\"utf-8\")\n            except UnicodeDecodeError:\n                reason = self.reason.decode(\"iso-8859-1\")\n        else:\n            reason = self.reason\n    \n        if 400 <= self.status_code < 500:\n            http_error_msg = (\n                f\"{self.status_code} Client Error: {reason} for url: {self.url}\"\n            )\n    \n        elif 500 <= self.status_code < 600:\n            http_error_msg = (\n                f\"{self.status_code} Server Error: {reason} for url: {self.url}\"\n            )\n    \n        if http_error_msg:\n>           raise HTTPError(http_error_msg, response=self)\nE           requests.exceptions.HTTPError: 400 Client Error:  for url: http://localhost:4001/users/signup\n\n/opt/hostedtoolcache/Python/3.12.1/x64/lib/python3.12/site-packages/requests/models.py:1021: HTTPError\n\nDuring handling of the above exception, another exception occurred:\n\nsign_up_api = <api.post_sign_up.SignUp object at 0x7ff6a1768f20>\n\n    def test_should_return_403_if_access_denied(sign_up_api: SignUp):\n        user = get_random_user()\n        user.token = \"invalid_token\"  # Simulate an invalid token or unauthorized access\n        try:\n            sign_up_api.api_call(user)\n        except requests.exceptions.HTTPError as e:\n>           assert e.response.status_code == 403, \"Expected status code 403\"\nE           AssertionError: Expected status code 403\nE           assert 400 == 403\nE            +  where 400 = <Response [400]>.status_code\nE            +    where <Response [400]> = HTTPError('400 Client Error:  for url: http://localhost:4001/users/signup').response\n\ntests/api/test_api_signup.py:58: AssertionError","flaky":false,"newFailed":true,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_session_faker","time":{"start":1718383430904,"stop":1718383430922,"duration":18},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"sign_up_api","time":{"start":1718383432713,"stop":1718383432713,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false}],"testStage":{"status":"failed","statusMessage":"AssertionError: Expected status code 403\nassert 400 == 403\n +  where 400 = <Response [400]>.status_code\n +    where <Response [400]> = HTTPError('400 Client Error:  for url: http://localhost:4001/users/signup').response","statusTrace":"sign_up_api = <api.post_sign_up.SignUp object at 0x7ff6a1768f20>\n\n    def test_should_return_403_if_access_denied(sign_up_api: SignUp):\n        user = get_random_user()\n        user.token = \"invalid_token\"  # Simulate an invalid token or unauthorized access\n        try:\n>           sign_up_api.api_call(user)\n\ntests/api/test_api_signup.py:56: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \napi/post_sign_up.py:7: in api_call\n    response = self.make_request(\"POST\", \"users/signup\", json=user_data)\napi/base_api.py:33: in make_request\n    response.raise_for_status()\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <Response [400]>\n\n    def raise_for_status(self):\n        \"\"\"Raises :class:`HTTPError`, if one occurred.\"\"\"\n    \n        http_error_msg = \"\"\n        if isinstance(self.reason, bytes):\n            # We attempt to decode utf-8 first because some servers\n            # choose to localize their reason strings. If the string\n            # isn't utf-8, we fall back to iso-8859-1 for all other\n            # encodings. (See PR #3538)\n            try:\n                reason = self.reason.decode(\"utf-8\")\n            except UnicodeDecodeError:\n                reason = self.reason.decode(\"iso-8859-1\")\n        else:\n            reason = self.reason\n    \n        if 400 <= self.status_code < 500:\n            http_error_msg = (\n                f\"{self.status_code} Client Error: {reason} for url: {self.url}\"\n            )\n    \n        elif 500 <= self.status_code < 600:\n            http_error_msg = (\n                f\"{self.status_code} Server Error: {reason} for url: {self.url}\"\n            )\n    \n        if http_error_msg:\n>           raise HTTPError(http_error_msg, response=self)\nE           requests.exceptions.HTTPError: 400 Client Error:  for url: http://localhost:4001/users/signup\n\n/opt/hostedtoolcache/Python/3.12.1/x64/lib/python3.12/site-packages/requests/models.py:1021: HTTPError\n\nDuring handling of the above exception, another exception occurred:\n\nsign_up_api = <api.post_sign_up.SignUp object at 0x7ff6a1768f20>\n\n    def test_should_return_403_if_access_denied(sign_up_api: SignUp):\n        user = get_random_user()\n        user.token = \"invalid_token\"  # Simulate an invalid token or unauthorized access\n        try:\n            sign_up_api.api_call(user)\n        except requests.exceptions.HTTPError as e:\n>           assert e.response.status_code == 403, \"Expected status code 403\"\nE           AssertionError: Expected status code 403\nE           assert 400 == 403\nE            +  where 400 = <Response [400]>.status_code\nE            +    where <Response [400]> = HTTPError('400 Client Error:  for url: http://localhost:4001/users/signup').response\n\ntests/api/test_api_signup.py:58: AssertionError","steps":[],"attachments":[{"uid":"6f2defa3abd0c217","name":"log","source":"6f2defa3abd0c217.txt","type":"text/plain","size":366}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":1,"hasContent":true},"afterStages":[],"labels":[{"name":"parentSuite","value":"tests.api"},{"name":"suite","value":"test_api_signup"},{"name":"host","value":"fv-az1433-185"},{"name":"thread","value":"3435-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.api.test_api_signup"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Product defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":1,"broken":0,"skipped":0,"passed":2,"unknown":0,"total":3},"items":[{"uid":"b5e5ddbcd72f8593","reportUrl":"https://michalcecula.github.io/python-e2e-localstack-tests/9//#testresult/b5e5ddbcd72f8593","status":"passed","time":{"start":1718382936500,"stop":1718382936806,"duration":306}},{"uid":"84daddb7cc0e45a6","reportUrl":"https://michalcecula.github.io/python-e2e-localstack-tests/7//#testresult/84daddb7cc0e45a6","status":"passed","time":{"start":1718382276089,"stop":1718382276404,"duration":315}}]},"tags":[]},"source":"c9f4c65a71a3d09.json","parameterValues":[]}